en/sensor/rangefinders.md

```diff
@@ -0,0 +1,183 @@
+
+The uLanding Radar is not present in "most" firmware by default and must be started by updating a configuration file (rather than by a parameter). More information can be found here: [uLanding Radar](../sensor/ulanding_radar.md).
+
+
+### Other
+
+PX4 also supports the Bebop rangefinder.
+
+
+## Rangefinder Configuration
+
+### Setup
+
+The rangefinder is configured using [EKF2\_RNG\_*](../advanced_config/parameter_reference.md#EKF2_RNG_POS_X) parameters. These configure information about the offset of the rangefinder from the centre of the vehicle body, the approximate delay of data reaching the estimator from the sensor, etc.
+
+### Enable/Use
+
+The rangefinder can be enabled in two ways:
+1. Set [EKF2_HGT_MODE](../advanced_config/parameter_reference.md#EKF2_HGT_MODE) to *Range finder* (`2`). This makes the rangefinder the primary source of height estimation (the default altitude sensor is the barometer).
+1. Set [EKF2_RNG_AID](../advanced_config/parameter_reference.md#EKF2_RNG_AID) to `1`. This makes the vehicle use the rangefinder as the primary source when it is safe to use, but will otherwise use the sensor specified in `EKF2_HGT_MODE`.
+   * Specifically, the rangefinder is enabled when: 
+     * velocity < [EKF2_RNG_A_VMAX](../advanced_config/parameter_reference.md#EKF2_RNG_A_VMAX)
+     * distance to ground < [EKF2_RNG_A_HMAX](../advanced_config/parameter_reference.md#EKF2_RNG_A_HMAX)
+   * Other parameters affecting "Range aid" are prefixed with `EKF2\_RNG\_A\_`.
+
+## Testing
+
+The easiest way to test the rangefinder is to vary the range and compare to the values detected by PX4. The sections below show some approaches to getting the measured range.
+
+### QGroundControl Analyze Tool 
+
+The *QGroundControl Analyze Tool* tool and *QGroundControl MAVLink Inspector* let you view messages sent from the vehicle, including `DISTANCE_SENSOR` information from the rangefinder. The main difference between the tools is that the *Analyze* tool can plot values in a graph.
+
+> **Note** The messages that are sent depend on the vehicle configuration. You will only get `DISTANCE_SENSOR` messages if the connected vehicle has a rangefinder installed and is publishing sensor values. 
+
+To view the rangefinder output:
+
+1. Open the menu **Widgets > Analyze**:
+
+   ![Menu for QGC Analyze Tool](../../assets/qgc/menu_analyze_tool.png)
+1. Select the message `DISTANCE_SENSOR.current_value`. The tool will then plot the result:
+   ![QGC Analyze DISTANCE_SENSOR value](../../assets/qgc/qgc_analyze_tool_distance_sensor.png)
+
+
+### QGroundControl MAVLink Console
+
+You can also use the *QGroundControl MAVLink Console* to observe the `distance_sensor` uORB topic:
+```sh
+listener distance_sensor 5
+```
+
+> **Note** This works when connected to Pixhawk or other NuttX targets, but not the Simulator. 

```

Comment:
Maybe we can mention that it does work in the terminal where you start the simulation

Reply:
Good catch. Done!